src/server/system_services/account_server.js

```diff
@@ -230,6 +230,23 @@ function read_account(req) {
 }
 
 
+function read_access_key_account(req) {
+    const { access_key } = req.rpc_params;
+
+    const account = _.find(system_store.data.accounts, acc =>
+        acc.access_keys && acc.access_keys[0].access_key.unwrap() === access_key.unwrap()
+    );
+
+    if (!account) {
+        throw new RpcError('NO_SUCH_ACCOUNT', 'No such account with credentials: ' + access_key);
```

Comment:
Adding the sensitive string access key to the error message is quite useless so either we unwrap or remove it.
Not sure if we really have to hide access_keys from the logs - @nimrod-becker is that needed?
```
        throw new RpcError('NO_SUCH_ACCOUNT', 'No such account for access key: ' + access_key.unwrap());
```

Reply:
I think we need to , otherwise each kubectl logs would reveal these